### ISSUE TITLE
Overfitting Yolov2 question

### ISSUE THREAD

**Issue by smallesan** [AlexeyAB/darknet#160]
I am trying to overfit the Yolov2 for specific items (50 in total). I have had some amount of success by training for an extended time, but is there any parameters that I can tune to make it overfit even better? Thanks.

**Comment by VanitarNordic** [comment]
increase the learning rate, use smaller dataset

**Comment by VanitarNordic** [comment]
Also please tell us why do you want to overfit the YOLO. Overfitting is a nightmare in deep learning models. is there any reason?

**Comment by gizmoy** [comment]
Yup, there is. Overfitting small portion of your dataset is a sanity check before you start 'heavy' training. If you are not able to overfit 10 examples something is probably wrong.

**Comment by git-sohib** [comment]
does anybody know how to overfit the yolov2 from scratch just to prove sanity checks are really helpful? Any links, posts are appreciated.
